nltk/tokenize/texttiling.py

```diff
@@ -312,7 +312,7 @@ def _depth_scores(self, scores):
         #that a section shouldn't be smaller than at least 2
         #pseudosentences for small texts and around 5 for larger ones.
 
-        clip = min(max(len(scores)/10, 2), 5)
+        clip = int(min(max(len(scores)/10, 2), 5))
```

Comment:
I would suggest to use integer division instead:
```python
clip = min(max(len(scores) // 10, 2), 5)
```

Reply:
If there is a case when the depth score is calculated incorrectly, it would be nice to include it as a doctest.